controllers/array_action/array_mediator_svc.py

```diff
@@ -97,6 +101,25 @@ def _is_space_efficiency_matches_source(parameter_space_efficiency, array_space_
            (parameter_space_efficiency and parameter_space_efficiency == array_space_efficiency)
 
 
+def build_create_volume_group_kwargs(name):
+    cli_kwargs = {
+        'name': name
+    }
+    return cli_kwargs
```

Comment:
isn't bit redundant? you can easily just send {'name': name} instead of calling this..
at least do `return {'name': name}`

Reply:
Maybe a bit, yes. I was thinking that maybe in the future we'll add other arguments.

Reply:
Done